app/bin/tools/datastore.dart

```diff
@@ -135,18 +135,14 @@ class UpdateCommand extends Command {
       exit(1);
     }
 
-    final backupDir = argResults['backup-directory'] == null
-        ? Directory.current
-        : new Directory(argResults['backup-directory'] as String);
-    // input files
-    final backupPrefix = await _detectLatest(backupDir);
-    print('\nFound backup prefix: $backupPrefix\n'
-        'Waiting for 5 seconds before proceeding.\n');
-    await new Future.delayed(const Duration(seconds: 5));
-    final packagesFile = new File('$backupPrefix-packages.jsonl');
+    final packagesFile = new File(argResults['packages'] as String);
+    if (!packagesFile.existsSync()) {
+      print('Packages file $packagesFile does not exists.');
+      exit(1);
+    }
 
     await withProdServices(() async {
-      print('Restoring packages from: $packagesFile');
+      print('Updating packages from: $packagesFile');
```

Comment:
'Restoring' might be more accurate than 'Updating'

Reply:
I've changed it to make it more consistent with the rest (e.g. command name, printed summary). Should I rename everything to restore?

Reply:
'restore' does make sense in a backup context and also suggests overwriting things. I think it makes sense, up to you.